database/reset.sh

```diff
@@ -0,0 +1,3 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
+psql -h $DB_HOST -p $DB_PORT -U $DB_USER $DB_NAME
```

Comment:
Should this run the schema file as it is intended to reset?

Reply:
Yes it should